src/coreclr/vm/ceemain.cpp

```diff
@@ -1835,42 +1835,57 @@ BOOL STDMETHODCALLTYPE EEDllMain( // TRUE on success, FALSE on error.
 
 struct TlsDestructionMonitor
 {
+    bool m_activated = false;
+
+    void Activate()
+    {
+        m_activated = true;
+    }
+
     ~TlsDestructionMonitor()
     {
         // Don't destroy threads here if we're in shutdown (shutdown will
```

Comment:
This comment does not look relevant. (It was not relevant even before this change.)